### PR TITLE
Add bech32-ibc to cosmos-sdk-proto-althea

### PR DESCRIPTION
**Pull request by ChristianBorst**

### CODE REVIEW

File: proto-build/src/main.rs [hunk 1]
```diff
@@ -19,14 +19,16 @@ use walkdir::WalkDir;
 // TODO(tarcieri): use a logger for this
 static QUIET: AtomicBool = AtomicBool::new(false);
 
+const BECH32IBC_REV: &str = "v0.2.0-rc2";
+
 /// The Cosmos SDK commit or tag to be cloned and used to build the proto files
-const COSMOS_SDK_REV: &str = "v0.44.1";
+const COSMOS_SDK_REV: &str = "1c4b62d70267374eb11f182ce1e97517fe7b4b56";
```
Comment (jkilpatr) [inline proto-build/src/main.rs]: why this specific rev rather than v0.44.2?

File: proto-build/src/main.rs [hunk 2]
```diff
@@ -147,6 +153,10 @@ fn update_submodules() {
     // be run once.
     run_git(&["submodule", "update", "--init"]);
 
+    info!("Updating osmosis-labs/bech32ibc submodule...");
+    run_git(&["-C", BECH32IBC_DIR, "fetch"]);
+    run_git(&["-C", BECH32IBC_DIR, "reset", "--hard", BECH32IBC_REV]);
```
Comment (jkilpatr) [inline proto-build/src/main.rs]: I preffered using libgit2 to submodules where you actually had rust bindings to git. But they ripped it out when 'upstreamed' oh well just griping.

**Review by jkilpatr — APPROVED** [review]
Needs a rebase and a few comments but looks good, increment the version and push to cargo

File: proto-build/src/main.rs (hunk 2 above)
Comment (ChristianBorst) [reply proto-build/src/main.rs]: :angry:

**Pull request merged**